snuba/datasets/pluggable_entity.py

```diff
@@ -88,13 +91,18 @@ def get_query_pipeline_builder(self) -> QueryPipelineBuilder[ClickhouseQueryPlan
         return SimplePipelineBuilder(query_plan_builder=query_plan_builder)
 
     def get_all_storages(self) -> Sequence[Storage]:
-        return [storage_and_mapper.storage for storage_and_mapper in self.storages]
+        return [storage_connection.storage for storage_connection in self.storages]
 
-    def get_all_storages_and_mappers(self) -> Sequence[StorageAndMappers]:
+    def get_all_storage_connections(self) -> Sequence[EntityStorageConnection]:
         return self.storages
 
     def get_writable_storage(self) -> Optional[WritableTableStorage]:
-        return self.writable_storage
+        for storage_connection in self.storages:
```

Comment:
could this use the storage_selector method `get_writable_storage_connection`? Seems to be the same thing

Reply:
See reply above